run-build-functions.sh

```diff
@@ -311,7 +311,7 @@ install_node() {
   fi
 
   # if Node.js Corepack is available enable it
-  if [ $(which corepack) ]; then
```

Comment:
Same missing `featureFlags` argument I believe.

Reply:
you are so right!